docs/user-guide/command-line-interface.md

```diff
@@ -357,6 +358,14 @@ Example:
 
     eslint --no-inline-config file.js
 
+#### `--print-config`
+
+This option outputs the configuration to be used for the files passed. All other options are ignored when present and no further processing will be done.
```

Comment:
"Files passed" -> "file passed"

Also, it's not true that all other options are ignored. Several of the options affect how the config is calculated. It's better to say that no linting is performed when using this option and only config-related options are valid in this mode.